src/OpenTelemetry.Exporter.Geneva/OpenTelemetry.Exporter.Geneva.csproj

```diff
@@ -0,0 +1,22 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <Description>An OpenTelemetry .NET exporter that exports to local ETW or UDS</Description>
+    <Authors>OpenTelemetry Authors</Authors>
+    <NoWarn>$(NoWarn),NU5104,CS1591,SA1123,SA1633,SA1310,CA1031,CA1810,CA1822,CA2000,CA2208,SA1204,SA1201,SA1202,SA1308,SA1309,SA1311,SA1402,SA1602,SA1649</NoWarn>
+    <TargetFrameworks>netstandard2.0</TargetFrameworks>
+    <TargetFrameworks Condition="$(OS) == 'Windows_NT'">$(TargetFrameworks);net461</TargetFrameworks>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="OpenTelemetry" Version="1.2.0-rc2" />
+  </ItemGroup>
+
+  <PropertyGroup>
+    <EnableNETAnalyzers>true</EnableNETAnalyzers>
+    <AnalysisLevel>latest</AnalysisLevel>
```

Comment:
Just FYI in this repo you should be able to just do...

https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/99967016d9cb3a6d76a95cae1c270f2bed5c6687/src/OpenTelemetry.Extensions/OpenTelemetry.Extensions.csproj#L9

Defined here:

https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/99967016d9cb3a6d76a95cae1c270f2bed5c6687/build/Common.targets#L3